src/actions/sources/loadSourceText.js

```diff
@@ -51,10 +51,12 @@ export function loadSourceText(source: SourceRecord) {
       return Promise.resolve();
     }
 
-    const telemetryStart = performance.now();
     const deferred = defer();
     requests.set(id, deferred.promise);
 
+    // The file is not loaded so we need to load it. This telemetry probe times
+    // the loading of the source file.
+    telemetry.start(loadSourceHistogram, source);
```

Comment:
I don't think we need the comment here, it kinda describes the action. Perhaps we can add a description to the jsdoc of the action